backend/prisma/schema.prisma

```diff
@@ -0,0 +1,16 @@
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Member {
+  id   String @id @default(uuid())
```

Comment:
Out of curiosity, what are the benefits of a UUID vs auto-incrementing int? I find UUIDs are less readable, we lose a somewhat-valuable default sort order, and they make the index less efficient. These are all academic points, mind you, and don't really affect us here.

Reply:
I just did it out of habit, we use UUID in all our other db tables

Reply:
We avoid auto-incrementing int's as this sort of behaviour means you get real problems with database scalability and replication. Its generally a good practice to avoid it wherever possible.